legate/tester/args.py

```diff
@@ -187,6 +187,39 @@
 )
 
 
+test_opts.add_argument(
+    "--cov-bin",
+    dest="cov_bin",
```

Comment:
argparse will compute this `dest` value automatically. There is no need to supply `dest` unless you want something different (here and below)

https://docs.python.org/3/library/argparse.html#dest